tests/view_tests/tests/test_debug.py

```diff
@@ -785,7 +787,7 @@ def test_reporting_of_nested_exceptions(self):
         self.assertEqual(
             2, html.count(implicit_exc.format("&lt;p&gt;Second exception&lt;/p&gt;"))
         )
-        self.assertEqual(10, html.count("&lt;p&gt;Final exception&lt;/p&gt;"))
+        self.assertEqual(11, html.count("&lt;p&gt;Final exception&lt;/p&gt;"))
```

Comment:
Of course Sarah, would love to.

This line checks the occurrences of the innermost exception's title in the HTML. One more of those was added as I introduced an additional ol tag with the class `context-summary-placeholder`. The string content `&lt;p&gt;Final exception&lt;/p&gt;` is rendered there via template insertion of the variable `frame.context_line`.

The addition of one more occurrence of an exception title actually applies to ALL exception titles. So the count for `&lt;p&gt;Second exception&lt;/p&gt;` and `&lt;p&gt;Top level&lt;/p&gt;` also increased by 1. But these are not tested here. Only the counts for messages which indicate the nesting of exceptions are tested some lines further up.

You can check the additional tags by comparing the 2 HTML contents. Hope I could answer your question and that everything makes sense :-)

Reply:
For clarification: The additional placeholder element with the class `context-summary-placeholder` was needed as the summary's substitute in the expanded state of an exception container. Because otherwise there was no way to keep the content of the summary between `pre-context` and `post-context` (it's position is always on the top of a details tag).